sleepylyze/nrem.py

```diff
@@ -1165,7 +1165,7 @@ def calc_spin_tstats(self):
         spindle_stats = pd.DataFrame(columns=columns)
         
         # fill dataframe
-        for chan in self.channels:
+        for chan in self.spindles:
```

Comment:
see prior comment about soft resets to help with undoing changes from past commits